### PR TITLE
Doc/add pipeline tutorial

### PR DESCRIPTION
**Pull request by apetkau**
Adds a tutorial for how to run the pipeline in IRIDA.

If you wish to test out the tutorial, you can download the fastq files from <https://www.ebi.ac.uk/ena/data/view/SRR8914694> and follow along in an IRIDA instance with the pipeline installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +39,70 @@ In order to use this pipeline, you will also have to install the [RGI][], [stara
...
 
-# Screenshots
+# Tutorial
 
-## Pipeline
+This tutorial shows how to run the **AMR Detection** pipeline in IRIDA.
+
+## Tutorial Data
+
+The pipeline requires as input paired-end sequence reads in FASTQ format. We will be using Illumina MiSeq data from a sample of **Campylobacter jejuni** for this tutorial with run ID `SRR8914694`. Paired-end fastq files can be downloaded from <https://www.ebi.ac.uk/ena/data/view/SRR8914694> (the **FASTQ files (FTP)** links). A collection additional example data can be found in the [IRIDA Sample Data][] package.
```
Comment (deepsidhu85) [inline README.md]: Add the word `of` after the word `collection` in the last sentence please

File: README.md [hunk 2]
```diff
@@ -23,18 +39,70 @@ In order to use this pipeline, you will also have to install the [RGI][], [stara
...
+
+In particular, you may want to modify the **Select a pointfinder organism** parameter, changing the value to `campylobacter`. If you which to completely disable searching the PointFinder database, you can instead set **Enable scanning for point mutations using the PointFinder database** to `false`.
+
+Once all your parameters are set, you can click the **Ready to Launch?** button to launch the pipeline.
+
+![launch.png][]
+
+## Monitoring Pipeline Status
+
+To monitor the status of the launched pipeline, please select the **Analysis > Your Analyses** menu.
```
Comment (deepsidhu85) [inline README.md]: Change `Analysis` to `Analyses`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Just a few minor text changes otherwise looks good

File: README.md (hunk 1 above)
Comment (apetkau) [reply README.md]: Done in f3744f7e7f81ccb10c32697136e0bd9322077ced

File: README.md (hunk 2 above)
Comment (apetkau) [reply README.md]: Done in f3744f7e7f81ccb10c32697136e0bd9322077ced

**Review by deepsidhu85 — APPROVED** [review]

**Comment by apetkau** [comment]
Thanks @deepsidhu85

**Comment by deepsidhu85** [comment]
> Thanks @deepsidhu85

You're welcome!

**Pull request merged**